ui/src/containers/FullActivitiesList/FullActivitiesInput.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import SingleInput from 'ui/components/AutoComplete2/Inputs/SingleInput/SingleInput';
+import OptionListItem from 'ui/components/OptionListItem';
+import { compose, withProps, defaultProps } from 'recompose';
+import { withModels } from 'ui/utils/hocs';
+import { fromJS, Map } from 'immutable';
+import { displayLangMap } from 'ui/utils/xapi';
+
+const renderOption = ({
+  useTooltip = false,
```

Comment:
Is this argument `useTooltip` necessary? `renderOption` is an internal function in this file and `useTooltip` looks to be always `true`.

Reply:
Outdated but not resolved.